tests/virtualization/universal/hotplugging_network_interfaces.pm

```diff
@@ -34,7 +34,7 @@ sub add_virtual_network_interface {
         if (get_var('VIRT_AUTOTEST') && is_kvm_host) {
             $interface_model_option = '--model virtio';
         }
-        script_retry("ssh root\@$guest ip l | grep " . $virt_autotest::common::guests{$guest}->{macaddress}, delay => 60, retry => 10, timeout => 60);
+        script_retry("ssh root\@$guest ip l", delay => 60, retry => 10, timeout => 60);
```

Comment:
Would you please explain why removing checking macaddress?

Reply:
Macaddress is created random after this changes, it is not declared in guest hash.

The test module works following this steps:
1. attach new interface
2. check if the new interface MAC exists
3. detach new interface and check the new MAC is removed
4. check the MAC of the original interface (this part was removed because original mac is not declared in guests hash)

I changed 4. because having SSH connection and the second network interface is confirm as removed (in 3.)  leaves only one possibility that the original interface is online.

Other possible solution is to get the new mac address and declare them upon guest install, this will make this check valid, but I don't think it is needed?

Reply:
I see. I agree with you that logically it will be fine if all goes in *expected* way. When something *unexpected* happens, will it be okay? Eg something like last time's weird thing of a second disk after adding a soft-fail for a bug?

Reply:
I just notice something else, the original mac address check is before starting the attachment of the new interface. In that position it seems even more strange.

Reply:
Ok, wait for your further decision here.

Reply:
I will reintroduce the check, it is a small change better be safe.

Reply:
I have added the mac address into the guest hash and currently running tests

15-5 xen http://openqa.qam.suse.cz/tests/79135 
12-5 xen http://openqa.qam.suse.cz/tests/79139 
15-6 kvm http://openqa.qam.suse.cz/tests/79138
12-5 kvm http://openqa.qam.suse.cz/tests/79141